MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+        </Grid>
+        <listView:SfListView x:Name="listView" ItemSize="70" SelectionMode="Single" AllowSwiping="True"
+    ItemSpacing="0,0,5,0" >
+            <listView:SfListView.StartSwipeTemplate>
+                <DataTemplate>
+                    <ViewCell>
+                        <ViewCell.View>
+                            <Grid BackgroundColor="SlateBlue"  HorizontalOptions="Fill" VerticalOptions="Fill">
+                                <Label Text="Start Swipe Template" TextColor="White" VerticalOptions="Center" HorizontalOptions="Center"/>
+                            </Grid>
+                        </ViewCell.View>
+                    </ViewCell>
+                </DataTemplate>
+            </listView:SfListView.StartSwipeTemplate>
+            <listView:SfListView.EndSwipeTemplate>
+                <DataTemplate>
+                    <ViewCell>
+                        <ViewCell.View>
+                            <Grid BackgroundColor="SlateBlue" HorizontalOptions="Fill" VerticalOptions="Fill">
+                                <Label Text="End Swipe Template" TextColor="White" VerticalOptions="Center"/>
+                            </Grid>
+                        </ViewCell.View>
+                    </ViewCell>
+                </DataTemplate>
+            </listView:SfListView.EndSwipeTemplate>
+        </listView:SfListView>
+    </StackLayout>
+</ContentPage.Content>
+{% endhighlight %}
+{% endtabs %}
+
+The `SwipeOffset` value should be positive for the left swiping of the listview item.
+ 
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void LeftSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], 200);
+}
+{% endhighlight%}
+{% endtabs %}
+
+The `SwipeOffset` value should be negative for the right-swiping of the listview item.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void RightSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], -150);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatic Start and End swiping in listview](Images/swiping/maui-listview-programmatic-swiping.gif)
+
+## Events
+
+### SwipeStarted Event
+
+The `SfListView.SwipeStarting` event is raised when the swipe offset changes from its initial value. 
+
+The `SwipeStarting` event provides the following properties in their arguments:
+
+ * `Index`: Defines the swiping item index.
+ * `DataItem`: Defines the underlying data associated with the swiped item as its arguments. 
+ * `Direction`: Defines the swipe direction of the swiped item.
+ 
+The `SwipeStarting` event is used for the following use case:
+
+ * You can cancel the swipe action for a particular item by setting the `Cancel` property of the `SwipeStartingEventArgs`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfListView x:Name="listView" ItemsSource="{Binding InboxInfo}" 
```

Comment:
add AllowSwiping API and ensure this in all places

Reply:
resolved